Eastern-Research-Group/mywaterway
# Feature/826 remove upstream zoom threshold

## Related Issues:
* [HMW-819](https://jira.epa.gov/browse/HMW-819)
* [HMW-826](https://jira.epa.gov/browse/HMW-826)
* [HMW-827](https://jira.epa.gov/browse/HMW-827)

## Main Changes:
* Updated the Upstream Watershed widget on the tribe page to remove the zoom threshold from the watersheds layer when the widget is active.
* Added a couple of tests for the Upstream Watershed widget.
* Added more non-attains tribes from the list provided by Jesse. I looked up the epaId and biaTribeCode from the tribal boundaries layer.
* Updated the eq download links to go against the assessments profile, instead of the assessmentUnits profile.

## Steps To Test:
1. Navigate to http://localhost:3000/tribe/FONDULAC
2. Open the layer list
3. Verify the Watersheds layer is grayed out
4. Click the upstream watershed
5. Verify the Watersheds layer shows on the map
6. Click somewhere on the map
7. Verify the upstream watershed is shown on the map
8. Turn the upstream watershed widget off again
9. Repeat steps 2 and 3
10. Run the new cypress tests and verify they pass
11. Navigate to the state/tribe page
12. Test out new non-attains tribes (note: wqp was down when I tested, so I just verified the tribe boundaries showed up)
13. Navigate to http://localhost:3000/community/dc/overview
14. Verify eq download links go against the assessments profile
